src/ui/server/test-mocks/index.ts

```diff
@@ -46,4 +47,4 @@ const mockRes = () => {
 
 const mockNext = jest.fn();
 
-export { mockAnswers, mockCisCountries, mockCountries, mockNext, mockSession, mockQuote, mockReq, mockRes };
+export { mockAnswers, mockApplication, mockCisCountries, mockCountries, mockNext, mockSession, mockQuote, mockReq, mockRes };
```

Comment:
Can we have each export on a new line? it's quite long for 1 row? Not important

Reply:
I prefer them on one line as well - this stems from the UKEF typescript starter package, don't think we should deviate from that?

Reply:
This will probably go into new lines anyway after a certain threshold, as we add more things to test-mocks